ClassTypes/PowerShell/EthicalHacking/Create-EthicalHackingLabplanAndLab.ps1

```diff
@@ -44,20 +57,23 @@ trap {
 #
 
 # Download AzLab module file, import, and then delete the file
+if (-not (Get-Module -ListAvailable -Name 'Az')) {
+    Import-Module Az -Force 
+}
 
-Import-Module Az.LabServices -Force
+$ClassName = "EthicalHacking"
 
 
 # Configure parameter names
-$rgName     = "$($ClassName)RG_" + (Get-Random)
-$labPlanName     = "$($ClassName)Acct_" + (Get-Random)
+$rgName     = "rg-$ClassName-$(Get-Random)"
+$labPlanName     = "lp-$ClassName$(Get-Random)"
 $labName    =  "$($ClassName)Lab"
 
 # Create resource group
 Write-Host "Creating resource group $rgName"
 $rg = New-AzResourceGroup -Name $rgName -Location $Location
```

Comment:
This spot does not need to be ToLower for RG name, we should probably just use the formatting as-is.  (just wanted to mention because of the comments below where we do need ToLower - this line doesn't need to change).

Reply:
Updated $rgName line to call .ToLower()